openapi/features-api.yaml

```diff
@@ -0,0 +1,1166 @@
+          description: Horse Racing - Belmont Stakes
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_horse_racing_kentucky_derby:
+          description: Horse Racing - Kentucky Derby
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_horse_racing_preakness_stakes:
+          description: Horse Racing - Preakness Stakes
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_ice_hockey:
+          description: Ice Hockey - (All)
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_ice_hockey_nhl:
+          description: Ice Hockey - NHL
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_mma:
+          description: Mixed Martial Arts - (All)
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_mma_ufc:
+          description: Mixed Martial Arts - UFC
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_soccer:
+          description: Soccer - (All)
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_soccer_concacaf_champions_league:
+          description: Soccer - CONCACAF Champions League
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_soccer_concacaf_gold_cup:
+          description: Soccer - CONCACAF Gold Cup
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_soccer_copa_america_men:
+          description: Soccer - COPA America Men's
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_soccer_fifa_world_cup_women:
+          description: Soccer - FIFA World Cup Women's
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_soccer_fifa_world_cup_men:
+          description: Soccer - FIFA World Cup Men's
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_soccer_mls:
+          description: Soccer - MLS
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_soccer_uefa_champions_league_men:
+          description: Soccer - UEFA Champions League Men's
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_softball:
+          description: Softball - (All)
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_softball_ncaa_women:
+          description: Softball - NCAA Women's
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_tennis:
+          description: Tennis - (All)
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_tennis_us_open:
+          description: Tennis - US Open
+          $ref: '#/components/schemas/StatsField'
+        phq_viewership_sports_tennis_wimbledon:
+          description: Tennis - Wimbledon
+          $ref: '#/components/schemas/StatsField'
+        phq_spend_conferences:
```

Comment:
Same as `phq_attendance_*` (bool or `StatsField`)

It's only `phq_rank` that's different.